README.md

```diff
@@ -47,6 +47,26 @@ npm test
 
 For detailed explanation on how things work, checkout the [guide](http://vuejs-templates.github.io/webpack/) and [docs for vue-loader](http://vuejs.github.io/vue-loader).
 
+## Run with Docker
+
+At a later stage we will make Docker images available via DockerHub. For now
+you can build/run as follows:
+
+``` bash
+docker build -t meggsimum/wegue:latest .
+
+docker run -it -p 8080:80 meggsimum/wegue:latest
+
+# in browser: open http:/127.0.0.1:8080
+
+```
+
+To run with a custom Wegue JSON config, use Docker Volume Mapping to override the default config:
+
+``` bash
+docker run -it -p 8080:80 -v $(pwd)/app-conf-mine.json:/usr/share/nginx/html/static/app-conf.json meggsimum/wegue:latest
```

Comment:
By mapping `app-conf-mine.json` to `static/app-conf.json` it will overwrite the default app-conf. By setting 

`-v $(pwd)/app-conf-mine.json:/usr/share/nginx/html/static/app-conf-mine.json` you could use both the dafault configs and the newly mounted one by http://localhost:8080/?appCtx=mine

Reply:
hold on, a double bash literal now in README.md text, will commit in few seconds....

Reply:
OK, totally fine with that. Always good to have several options.